jena-core/src/main/java/org/apache/jena/datatypes/xsd/XSDDatatype.java

```diff
@@ -124,9 +121,6 @@ public class XSDDatatype extends BaseDatatype {
     public static final XSDDatatype XSDName = new XSDBaseStringType("Name");
 
     /** Datatype representing xsd:QName */
-    // If you see this, remove commented lines.
-    // Merely temporary during switch over and testing.
-    // public static final XSDDatatype XSDQName = new XSDDatatype("QName");
     public static final XSDDatatype XSDQName = new XSDPlainType("QName");
```

Comment:
Lets' add `@deprecated(forRemoval = true)` and `/** @deprected Do not use */` to `XSDQname` and the other non-RDF XSD datatypes.

Reply:
Other than that, the rest looks great!

Reply:
Done. I've also added a note to remove these in Jena 6, together with the associated dummy code in BaseSchemaDVFactory.